### PR TITLE
Various fixes from Scav issue list.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
- Temporarily prevents https://github.com/ScavStation/ScavStation/issues/903 from occurring pending a proper fix/rewrite.
- Fixes https://github.com/ScavStation/ScavStation/issues/904
- Fixes https://github.com/ScavStation/ScavStation/issues/901
- Fixes https://github.com/ScavStation/ScavStation/issues/900
- Fixes https://github.com/ScavStation/ScavStation/issues/902
- Fixes frame icons in https://github.com/NebulaSS13/Nebula/issues/3736, can't repro the pilot ghosts.
- Fixes https://github.com/ScavStation/ScavStation/issues/909.
image

### CODE REVIEW

File: code/modules/multiz/level_data.dm [hunk 1]
```diff
@@ -269,6 +273,13 @@
 		var/list/possible_strata = list()
 
 		for(var/stype in all_strata)
+			var/forbidden = FALSE
+			for(var/forbid_type in forbid_strata)
```
Comment (comma) [inline code/modules/multiz/level_data.dm]: there's `is_type_in_list(datum/thing, list/type_list)`

File: code/modules/reagents/reagent_containers/drinkingglass/drinkingglass.dm [hunk 2]
```diff
@@ -12,32 +12,34 @@ var/global/const/DRINK_ICON_NOISY = "noise"
...
-	var/static/list/filling_icons_cache = list()
-
 	center_of_mass =@"{'x':16,'y':9}"
-
 	amount_per_transfer_from_this = 5
 	possible_transfer_amounts = @"[5,10,15,30]"
 	atom_flags = ATOM_FLAG_OPEN_CONTAINER
 	presentation_flags = PRESENTATION_FLAG_NAME | PRESENTATION_FLAG_DESC
 	temperature_coefficient = 4
-	obj_flags = OBJ_FLAG_HOLLOW
```
Comment (comma) [inline code/modules/reagents/reagent_containers/drinkingglass/drinkingglass.dm:36]: Did you mean to remove the flag?

File: code/modules/reagents/reagent_containers/pill.dm [hunk 3]
```diff
@@ -93,6 +105,7 @@
 
 /obj/item/chems/pill/bromide/populate_reagents()
 	reagents.add_reagent(/decl/material/liquid/bromide, reagents.maximum_volume)
+	. = ..()
 
 /obj/item/chems/pill/cyanide
```
Comment (comma) [inline code/modules/reagents/reagent_containers/pill.dm:109]: this probably shouldn't autolabel

File: code/modules/reagents/reagent_containers/pill.dm [hunk 4]
```diff
@@ -175,6 +197,7 @@
 
 /obj/item/chems/pill/brute_meds/populate_reagents()
 	reagents.add_reagent(/decl/material/liquid/brute_meds, 20)
+	. = ..()
 
 /obj/item/chems/pill/happy
 	name = "happy pill"
```
Comment (comma) [inline code/modules/reagents/reagent_containers/pill.dm:196]: ditto, same for zoom and gleam below

File: code/modules/reagents/reagent_containers/drinkingglass/drinkingglass.dm (hunk 2 above)
Comment (MistakeNot4892) [reply code/modules/reagents/reagent_containers/drinkingglass/drinkingglass.dm:36]: Yeah it's redundant, set on the lower type.

File: code/modules/multiz/level_data.dm [hunk 5]
```diff
@@ -108,6 +108,10 @@
 	var/decl/strata/strata
 	///The base material randomly chosen from the strata for this level.
 	var/decl/material/strata_base_material
+	///Strata types to forbid from generating on this level.
+	var/list/forbid_strata = list(
+		/decl/strata/permafrost
```
Comment (out-of-phaze) [inline code/modules/multiz/level_data.dm:113]: Why is this necessary? Is the maximum temperature check still broken on stable somehow?

**Review by out-of-phaze — DISMISSED** [review]

File: code/modules/multiz/level_data.dm (hunk 5 above)
Comment (MistakeNot4892) [reply code/modules/multiz/level_data.dm:113]: No. Ice spawning as the only strata for space mining levels like Scav uses is just basically worthless for the regular SS13 mining loop.

Comment (out-of-phaze) [reply code/modules/multiz/level_data.dm:113]: Oh, right, space levels. Yeah, that checks out. Permafrost kind of sucks, it's just gases and the ways to harvest them are kind of buggy/incomplete.

**Review by out-of-phaze — APPROVED** [review]
